bahmnicore-omod/src/main/resources/liquibase.xml

```diff
@@ -4523,28 +4523,22 @@
     		INSERT INTO role_privilege (role, privilege) VALUES ('Anonymous', 'Get Locations');
     	</sql>
     </changeSet>
-    <changeSet id="bahmni-core-202011150948" author="Buvaneswari">
+    <changeSet id="bahmni-core-20201210148" author="Buvaneswari">
         <preConditions onFail="MARK_RAN">
             <not><tableExists tableName="form_privilege"/></not>
         </preConditions>
         <comment>Introducing form_privilege table as part of BAH_790 Privilege based access to forms</comment>
         <sql>
-            CREATE TABLE form_privilege (
+            CREATE TABLE form_privilege(
             form_privilege_id int(11) NOT NULL AUTO_INCREMENT,
             formId int(11) NOT NULL DEFAULT '0',
             formVersion varchar(50) NOT NULL DEFAULT '0',
             privilegeName varchar(50) NOT NULL DEFAULT '',
             editable tinyint(1) DEFAULT '0',
             viewable tinyint(1) DEFAULT '0',
-            creator int(11) NOT NULL DEFAULT '0',
             date_created datetime NOT NULL,
             date_changed datetime DEFAULT NULL,
-            changed_by int(11) DEFAULT NULL,
```

Comment:
Are we deleting the entry or voiding the entry in the form_privilege table? If we are voiding the entry, then we must have the "changed_by" column.

Reply:
We are not voiding but deleting the entry.

Reply:
We dont need this PR anymore right? please close it

Reply:
@buvaneswari-arun ^

Reply:
BAH-1115 is created for this activity.